egs/librispeech/asr/simple_v1/mmi_bigram_decode.py

```diff
@@ -236,7 +236,7 @@ def main():
             L = k2.Fsa.from_openfst(f.read(), acceptor=False)
         logging.debug("Loading G.fsa.txt")
         with open(lang_dir / 'G.fsa.txt') as f:
-            G = k2.Fsa.from_openfst(f.read(), acceptor=True)
+            G = k2.Fsa.from_openfst(f.read(), acceptor=False)
```

Comment:
what's the aux/output labels of G now?

Reply:
Some arcs contain #0:eps

Reply:
I'm confused here, so when you do `compose(L,G)` below, the output labels are `eps`? Anyway, it's strange to have aux_labels on G here..

Reply:
I would recommend you to run kaldi's arpa2fst on a simple arpa file and see whether the resulting G.fst is an acceptor
or a transducer.

Reply:
....you did not answer me about the question of `compose(L,G)`....what's the aux_labels on its result?

Reply:
If L.aux_labels contains `#0`, then the LG.aux_labels may contain `eps` if an arc with `#0:eps` in G is used.

Are there any problems if LG.aux_labels contain eps?

Reply:
> I'm confused here, so when you do compose(L,G) below, the output labels are eps? 

Can you find an example that when you do `compose(L,G)`, the `result.aux_labels` contains no `eps`?
(Assume a word has at least two phones)

Reply:
ok, you mean normal word  and eps...so i think we can remove below code that doing disambig removing on  aux labels

Reply:
When we are preparing LG for training (without disambig symbols) I think you could project G on the output: meaning, set the labels to the aux_labels, and then remove the aux_labels if you want.

Reply:
... for me it's very odd if compose(L, G) is not terminating.  If anything should fail, I'd expect it to be the determinization.